recipe/meta.yaml

```diff
@@ -25,8 +25,8 @@ build:
   number: 3
   string: h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
   ignore_run_exports_from:
-    - python
-    - nodejs
+    - python *
+    - nodejs *
```

Comment:
Thanks for the help. I've tried that before, and it didn't actually ignore the run-exports. See
https://github.com/conda-forge/emscripten-feedstock/blob/b0458c0409635e2b60cb48c95831373a0ce5072f/recipe/meta.yaml#L25-L30
and yet the `_0` builds [depend](https://conda-metadata-app.streamlit.app/?q=conda-forge%2Flinux-64%2Femscripten-3.1.58-py39h84d6215_0.conda) on `python >=3.9,<3.10.0a0`